Sources/TuistGenerator/Mappers/ResourcesProjectMapper.swift

```diff
@@ -56,7 +56,11 @@ public class ResourcesProjectMapper: ProjectMapping {
         }
 
         if target.supportsSources {
-            let (filePath, data) = synthesizedFile(bundleName: bundleName, target: target, project: project)
+            let (filePath, data) = synthesizedFile(
+                bundleName: bundleName.replacingOccurrences(of: "-", with: "_"),
```

Comment:
If the parameter is called `bundleName`, then we should pass the `bundleName`. If we need to do any replacement it is a concern of the `synthesizedFile` method